orc8r/cloud/deploy/terraform/orc8r-helm-aws/variables.tf

```diff
@@ -75,6 +75,12 @@ variable "monitoring_kubernetes_namespace" {
   default     = "monitoring"
 }
 
+variable "orc8r_staging_namespace" {
```

Comment:
Should we consider adding a enable_blue_green variable and use that as a condition to create the namespace with the default value set to a non-empty string of our preference? This way, the secondary namespace will be created if customer using blue-green and will have a default value already. 

Might be a better alternative to consider over the check for empty vs valid string.